README.md

```diff
@@ -268,7 +268,7 @@ that will help us track down the bug faster.
 ### Submitting a Feature Request
 
 For feature requests, please feel free to file a
-[GitHub issue](https://github.com/apple/swift-docc/issues/new?template=FEATURE_REQUEST.md)
+[GitHub issue](https://github.com/apple/swift-docc/issues/new?assignees=&labels=enhancement&template=FEATURE_REQUEST.yml)
```

Comment:
This label doesn't seem to exist in this repo. Did you mean to use 'improvement'?

Reply:
That's what the user will direct to when they click "Issues" -> "New Issue" -> "Feature Request Get Started".

We should add this label or change the label defined here https://github.com/apple/swift-docc/blob/main/.github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml

Reply:
This label for FEATURE_REQUEST issue template is always "enhancement". #268 just adopts the Github's new Form format and did not change it.